.env.example

```diff
@@ -16,9 +21,6 @@ LIVEKIT_URL=wss://my-livekit-project.livekit.cloud
 # S3_REGION=
 
 # PUBLIC
-NEXT_PUBLIC_LK_TOKEN_ENDPOINT=/api/token
-#NEXT_PUBLIC_LK_RECORD_ENDPOINT=/api/record
```

Comment:
why was the record endpoint removed entirely?

Reply:
Exidentially removed. Good catch!